python/pyspark/sql/tests.py

```diff
@@ -1901,7 +1901,7 @@ def test_list_tables(self):
         self.assertEquals(len(tables), 2)
         self.assertEquals(len(tablesSomeDb), 2)
         self.assertEquals(tables[0], Table(
-            name="tab1",
+            name="t1",
```

Comment:
python test failed 
```
the location('file:/home/jenkins/workspace/SparkPullRequestBuilder/spark-warehouse/tab1') of table('`default`.`tab1`') already exists.;
```
the location of tab1 does not deleted , I have tried to found out in which test case it forget to delete it(search all test cases containing `tab1` and run it), but they are all ok to delete the location of tab1, so here we change the table name to work around it.